packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -32,6 +33,15 @@ const getActiveRouteName = (state: NavigationState) => {
 
 const RESTORE_STATE = __DEV__ && DEV_RESTORE_NAV_STATE_ON_RELOAD
 
+// Global app them used by react-navigation
+const AppTheme = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    background: colors.light,
+  },
+}
+
```

Comment:
This is where we define the default background color for screens.

Reply:
cool.

also just noticed you can comment on multiple lines. nice